MANIFEST/deeplab_v3.manifest.yaml

```diff
@@ -0,0 +1,4 @@
+COMMAND:
+  - name: DeepLabV3 Image Segmentation
+    key: DEEPLAB_V3
+    type: SEGMENTATION
```

Comment:
please add a newline

Reply:
Not sure if you want it at the head or tail, please use "Add a suggestion"